# 20 SPR flow

Is the holdout correlation between the weekly SPR stock change and the Wednesday RWTC return negative?

Holdout starts 2020-01-01. Verdict: failed.

Weekly change in EIA WCSSTUS1, the Strategic Petroleum Reserve, thousand barrels. Kept only when the two week-ending dates are 7 days apart. A negative change is a draw.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'spr-flow')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


failed
Train correlation of the SPR change with the Wednesday RWTC return is 0.019. Holdout correlation is 0.047. The holdout correlation is not negative. The bar was missed.
WCSSTUS1 weekly change versus the Wednesday RWTC log return
Train | 1584 | 0.019
Holdout | 348 | 0.047
Skipped, no Wednesday RWTC print | 359 | 


## Method

The bar is a negative holdout correlation: a draw (negative change) lining up with a higher Wednesday return.

## Result

Train correlation of the SPR change with the Wednesday RWTC return is 0.019. Holdout correlation is 0.047. The holdout correlation is not negative. The bar was missed.